### PR TITLE
En_Fu_Mato

### PR DESCRIPTION
**Pull request by Kelebek1**
Before opening this PR, ensure the following:
- `./format.sh` was run to apply standard formatting.
- `make` successfully builds a matching ROM.
- No new compiler warnings were introduced during the build process.
    - Can be verified locally by running `tools/warnings_count/check_new_warnings.sh`
- New variables & functions should follow standard naming conventions.
- Comments and variables have correct spelling.
---

### CODE REVIEW

File: src/overlays/actors/ovl_En_Fu/z_en_fu.h [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #define Z_EN_FU_H
 
 #include "global.h"
+#include "overlays/actors/ovl_Bg_Fu_Mizu/z_bg_fu_mizu.h"
```
[the remark this answers is not in the document]
Comment (Kelebek1) [reply src/overlays/actors/ovl_En_Fu/z_en_fu.h:5]: It's used in the header,a nd I got an error after importing this header that BgFuMizu is undefined. I don't even know why it worked in the first place...

[the remark this answers is not in the document]
Comment (AngheloAlf) [reply src/overlays/actors/ovl_En_Fu/z_en_fu.h:5]: This kind of thing happens when the decomper doesn't include the headers properly

[the remark this answers is not in the document]
Comment (hensldm) [reply src/overlays/actors/ovl_En_Fu/z_en_fu.h:5]: Since your including this in the `.h` file now, you should delete the include from the `.c` file.

**Review by hensldm — APPROVED** [review]

File: src/overlays/actors/ovl_En_Fu/z_en_fu.c [hunk 2]
```diff
@@ -6,7 +6,6 @@
 
 #include "overlays/actors/ovl_En_Fu_Mato/z_en_fu_mato.h"
 #include "overlays/actors/ovl_En_Fu_Kago/z_en_fu_kago.h"
-#include "overlays/actors/ovl_Bg_Fu_Mizu/z_bg_fu_mizu.h"
 #include "overlays/actors/ovl_Bg_Fu_Kaiten/z_bg_fu_kaiten.h"
 #include "overlays/actors/ovl_En_Bom/z_en_bom.h"
 #include "z_en_fu.h"
```
Comment (AngheloAlf) [inline src/overlays/actors/ovl_En_Fu/z_en_fu.c:11]: Could you reorder the includes to put the en_fu one on top?

**Review by AngheloAlf — APPROVED** [review]

**Review by Kenix3 — APPROVED** [review]

**Pull request merged**